claude_scripts/claude_md_updater.py

```
@@ -0,0 +1,329 @@
+        shutil.copy2(CLAUDE_MD_PATH, backup_path)
+        return backup_path
+
+    def get_promotable_patterns(self):
+        """Get patterns ready for promotion to CLAUDE.md"""
+        if not MEMORY_FILE.exists():
+            return []
+
+        try:
+            with open(MEMORY_FILE) as f:
+                memory = json.load(f)
+        except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)